docs/book/platform-guide/set-up-your-mlops-platform/set-up-your-mlops-platform.md

```diff
@@ -32,51 +36,66 @@ See the following section to learn about the different ways of [deploying ZenML]
 
 The Compute infrastructure (e.g. Kubernetes or serverless alternatives)
```

Comment:
Feels like this sentence is not completed.

Reply:
True